app/build.gradle

```diff
@@ -0,0 +1,36 @@
+apply plugin: 'com.android.application'
+
+android {
+    compileSdkVersion 23
+    buildToolsVersion '25.0.0'
+
+    defaultConfig {
+        minSdkVersion 17
+        targetSdkVersion 23
```

Comment:
If I am not mistaken targetSdkVersion needs to be 25 if we want this for nougat as well